client/src/App.jsx

```diff
@@ -7,26 +7,26 @@ import {
   Route,
   Link,
 } from "react-router-dom";
-import Foot from "../src/components/Footer/Foot.tsx";
+import Foot from "../src/components/Header/Header.tsx";
 import "./App.css";
 
-import HomePage from './components/HomePage/HomePage.tsx';
-import Projects from './components/Projects/Projects.tsx';
-import Events from './components/Events/Events.tsx';
+import HomePage from "./components/HomePage/HomePage.tsx";
+import Projects from "./components/Projects/Projects.tsx";
+import Events from "./components/Events/Events.tsx";
 
 // Add new paths here!
 const router = createBrowserRouter([
   { path: "/", element: <HomePage /> },
   { path: "/projects", element: <Projects /> },
-  { path: "/events", element: <Events />},
+  { path: "/events", element: <Events /> },
 ]);
 
 function App() {
   return (
     <>
       <Foot />
-      <section style={{ marginTop: '10%'}}>
-      <RouterProvider router={router} />
+      <section style={{ marginTop: "10%" }}>
```

Comment:
Would recommend setting this to a set value (ex: 120px) to prevent subpages from colliding with the header